modules/apps/forms-and-workflow/calendar/calendar-web/src/main/resources/META-INF/resources/js/calendar_container.js

```diff
@@ -223,7 +223,12 @@ AUI.add(
 											if (calendarResourceId == instance.get('groupCalendarResourceId') && item.get('permissions').MANAGE_BOOKINGS) {
 												defaultCalendar = item;
 											}
-											else if (calendarResourceId == instance.get('userCalendarResourceId') && defaultCalendar == null) {
+
```

Comment:
Hi @ricardocousolr,

There is any specific reason for not use only one `if` to check if the `defaultCalendar == null`? I imagine you desire to keep the same code pattern used in the method `CalendarDisplayContext.getDefaulCalendar`, am I right?